03-Node-npm-modules/ver.0.2/downloableMovie v0.2.js

```diff
@@ -0,0 +1,107 @@
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+
+var Movie = function (optParams) {
+  this.title = optParams.title;
+  this.duration = optParams.duration; 
+  this.director = optParams.director;
+  this.observer = new MovieObserver();
+};
+
+Movie.prototype = {
+  constructor : Movie,
+
+  getTitle : function () {
+     console.log(this.title);
+  }, 
+
+  setTitle :function (argTitle) {
+    this.title = argTitle;
+      
+  },
+  getDuration : function () {
+    console.log(this.duration);
+  }, 
+
+  setDuration :function (argDuration) {
```

Comment:
most of the variables do not need to be prefixed with arg, or param, or something like this representing which kind of thing it stores. With duration is just fine :)